NEWS.md

```diff
@@ -6,6 +6,7 @@
 to increase sampling efficiency thanks to Andrew Johnson. (#1017, #1053)
 * Enable `posterior_predict` for truncated continuous models
 even if the required CDF or quantile functions are unavailable.
+* Add support for within-chain threading with `rstan (Stan >= 2.25)` backend.
```

Comment:
This should be Stan >= 2.26 as the next release of rstan on CRAN will be with Stan 2.26. There will not be a 2.25 release. Also change that in the code.

Reply:
Thanks! I'll update it once `Stan 2.26` is released or at least when the version number is bumped on the `develop` branch. Right now, the development version of `rstan` uses `Stan 2.25`.